### PR TITLE
Fix modifying settings modifies any phase

### PR DESCRIPTION
**Pull request by chengsieuly**

### CODE REVIEW

File: app/components/Settings/settings.js [hunk 1]
```diff
@@ -12,12 +12,15 @@ export default class Settings extends PureComponent {
...
   }
 
-  setNewTime(val, mins, secs) {
-    mins(val);
-    secs(0);
+  setNewTime(val, mins, secs, phase) {
+    const { currentPhase } = this.props;
+    if (currentPhase === phase) {
+      mins(val);
+      secs(0);
```
Comment (VDUCHEW) [inline app/components/Settings/settings.js]: Did you mean to pass phase inside secs()?

Comment (chengsieuly) [inline app/components/Settings/settings.js]: Oh no, phase tells us what `phase` we're on so essentially, when someone edits a settings, i'm also passing in what phase they're trying to modify. Then, we can check if the current phase they're on is the same phase that they're editing. If so, we will reset mins and secs. That was the thinking

Comment (VDUCHEW) [inline app/components/Settings/settings.js]: I just realized that after looking at currentPhase. LGTM

File: app/components/Settings/settings.js [hunk 2]
```diff
@@ -45,10 +46,13 @@ export default class Settings extends PureComponent {
...
-          onChange={(val) => {
-            this.onSettingsChange('rounds.focusLength', val,
-            setFocusLength, this.setNewTime(val, setMinutes, setSeconds));
-          }}
+          onChange={val =>
+            this.onSettingsChange(
+              'rounds.focusLength',
+              val,
+              setFocusLength,
+              this.setNewTime(val, setMinutes, setSeconds, Phases.FOCUS)
```
Comment (chengsieuly) [inline app/components/Settings/settings.js]: made some edits to make it extra clear! @VDUCHEW

**Pull request merged**